app/authenticated/cluster/index/route.js

```diff
@@ -10,6 +10,7 @@ export default Route.extend({
       const cluster = this.get('scope').currentCluster;
 
       return {
+        description: cluster.description,
```

Comment:
This already exists so you didn't cause it but this is another place where the route is returning non-live data that won't auto update if it changes... return the `cluster` itself and then show `model.cluster.description` instead.

Also if possible (I'm not sure it has all the same data?) use `machines` instead of `nodes` for the dashboard info.  We're trying to not expose 2 slightly different types for these to the public API.  The `machines` of a `cluster` are also a computed property on `cluster` so you don't need to do anything special in the route to get them.